cpp/src/reductions/nested_type_minmax_util.cuh

```diff
@@ -42,7 +42,7 @@ struct row_arg_minmax_fn {
   }
 
   // This function is explicitly prevented from inlining, because it calls to
-  // `row_lexicographic_comparator::operator()` which is inlined and very heavy-weight. As a result,
+  // `DeviceComparator::operator()` which is inlined and very heavy-weight. Otherwise,
   // instantiating this functor will result in huge code, and objects of this functor used with
   // `thrust::reduce_by_key` or `thrust::scan_by_key` will result in significant compile time.
```

Comment:
The wording is a little stilted, and not much better than `As a result`. It isn't obvious what `Otherwise` is referring to.
```suggestion
  // `DeviceComparator::operator()` which is inlined and very heavy-weight. Inlining
  // this functor would result in huge code when instantiated, and significantly increase
  // compile time, if used with `thrust::reduce_by_key` or `thrust::scan_by_key`.
```